crates/sui-indexer/src/restorer/formal_snapshot.rs

```diff
@@ -119,8 +115,30 @@ impl IndexerFormalSnapshotRestorer {
         )
         .await?;
         info!("Finished restoring move objects");
-        self.restore_display_table().await?;
-        info!("Finished restoring display table");
+        Ok(())
+    }
+
+    // last checkpoint for watermark and chain identifier
+    async fn restore_checkpoints(&self) -> Result<(), IndexerError> {
```

Comment:
This could probably have a clearer name, We're not really restoring "checkpoints" but rather the chain identifier and the final checkpoint in the epoch we're storing from.